src/components/Header/HeaderUserMenu/HeaderUserMenu.js

```diff
@@ -9,11 +9,11 @@ const HeaderUserMenu = ({ user }) => {
 
   return (
     <div>
-      <span className="user-menu--open">
+      <a href="#" className="user-menu--open">
```

Comment:
If I am not wrong, this is not a good practice: if it is not a real link, it should be a button.

Reply:
Yes!!! Thanks!